### DISCUSSION TITLE
Invoice and related details management software in Budibase

### DISCUSSION THREAD

**Discussion by AKElements** [Budibase/budibase#12419] [Help] [answered]
Hello everyone, 
I'm quite a newbie so please just try to be patient,
I'm building the following software in budibase and I need a couple things to make it work. I think the first part is the most difficult since next pages need to insert data manually. In the upper part I need to assign automatically an ID at the creation  of  a new row ( which is in the "preventivi" screen) based on the lowest ID found. I need the lower part rows to be linked to one row in the upper part ( first table)  based on the ID (I've already created a relationship for that, I just need to set filters so that the only ones shown are the ones with the ID matching to the ID in the first table). I also need the Elimina= Delete button to delete all the checked tables (checkbox is a column which whenever true I'd like to delete by pressing delete). Couple more things further on, planning on paying someone also to teach in case it's needed. thank you in advance.
I'll attacch a excel file of how it should work with some changes and also screenshots of the software so far.
[FILE PROGRAMMA .it.en.xlsx](https://github.com/Budibase/budibase/files/13432250/FILE.PROGRAMMA.it.en.xlsx)
![Screenshot 2023-11-22 012434](https://github.com/Budibase/budibase/assets/151386484/b695448a-495d-4508-868d-10a17672fdc0)
![Screenshot 2023-11-22 013153](https://github.com/Budibase/budibase/assets/151386484/67ade237-a0cf-4b02-be65-4db7496a8a53)

**Comment by melohagan** [comment, marked as answer]
Hey @AKElements 

If there's a more specific question you have I think I may be able to give you a more helpful answer.

You can delete selected table rows like so:
![Screenshot 2023-11-28 at 16 12 48](https://github.com/Budibase/budibase/assets/101575380/cdd82eaf-618a-4212-bc43-aa0459fcddbe)

`{{ Table.Selected rows }}`

![Screenshot 2023-11-28 at 16 13 23](https://github.com/Budibase/budibase/assets/101575380/3f2ceb67-d630-442d-a1d3-a03efda29abb)
You can find the selected rows binding in the binding drawer when you click the lightning bolt
